.neuro/live.yml

```diff
@@ -48,3 +54,13 @@ jobs:
     action: gh:neuro-actions/filebrowser@v1.0.0
     args:
       volumes_project_remote: $[[ volumes.project.remote ]]
+
+  jupyter:
+    action: gh:neuro-actions/jupyter@master
+    args:
+      image: neuromation/base:v1.7.11
+      volumes_data_remote: $[[ volumes.data.remote ]]
+      volumes_code_remote: $[[ volumes.code.remote ]]
+      volumes_config_remote: $[[ volumes.config.remote ]]
+      volumes_notebooks_remote: $[[ volumes.notebooks.remote ]]
```

Comment:
```suggestion
      volumes_notebooks_remote: $[[ upload(volumes.notebooks).remote ]]
```

Reply:
It could be dangerous to use this option for notebooks, because you edit notebooks in browser - that means these changes aren't local and finally, if you don't download your changes, they will be lost (in contrast to code and config, which are edited locally)

Reply:
actually not, the `upload` command [utilizes](https://github.com/neuro-inc/neuro-flow/blob/master/neuro_flow/expr.py#L261) `neuro cp -u` option, which is the update. It means, that if the file version (modification time) on storage is newer - it woun't rewrite it (checkout `neuro cp --help`)

Reply:
OK, got it. Done.